### DISCUSSION TITLE
Purging unused CSS from AntD in NextJS

### DISCUSSION THREAD

**Discussion by philmetzger** [ant-design/ant-design#29690] [Q&A]
Hello,

I am trying to purge css from AntD but struggling to figure out how to do it.
In my NextJS project, in the `_app.js` I import the following: `import 'antd/dist/antd.min.css';`

Now i was looking at the purgeCSS docs for antD and saw this: https://purgecss.com/ant_design.html
This is the code from the link above:

```
const glob = require("glob-all");
const paths = require("react-scripts/config/paths");

const { override, addPostcssPlugins } = require("customize-cra");

const purgecss = require("@fullhuman/postcss-purgecss")({
  content: [
    paths.appHtml,
    ...glob.sync(`${paths.appSrc}/**/*.js`, { nodir: true }),
    ...glob.sync(`${paths.appNodeModules}/antd/es/button/**/*.css`, {
      nodir: true,
    }),
  ],
  extractors: [
    {
      extractor: (content) => content.match(/([a-zA-Z-]+)(?= {)/g) || [],
      extensions: ["css"],
    },
  ],
});

module.exports = override(
  addPostcssPlugins([
    ...(process.env.NODE_ENV === "production" ? [purgecss] : []),
  ])
);
```

What I am not sure about is where does this go? 
Is this the way to go about purging unused css from antD?

**Comment by quantuminformation** [comment]
Anyone know how to pause the CSS with Vite?
